Docs/feedback_design.md

```diff
@@ -1,29 +1,129 @@
-## Collected `az next` feedback data
+## Collected `az next` and `az interactive` feedback data
 
 ### Storage
 Collected by Telemetry
 
 ### Data scheme
-|   Field   |   Content   |   Value   |
-|  -------  |  ---------  |  -------  |
-| request_type | The recommendation type that the user filters | 1:all 2:solution 3:command 4:scenario |
-|  option | Options adopted by users | -1:no recommendation 0:not adopted >0:adopted |
-| latest_command | The last command that triggers the recommendation | command name |
-| error_info | Error infos when using the recommendation of Solution | error info |
-| rec_source | The data source of recommendation items | space separated array (item value: 1:knowledge base 2:caculation 3:Aladdin) |
-| rec_type | The type of recommendation items | space separated array (item value: 2:solution 3:command 4:scenario) |
-| adoption_source | The data source of adoption | 1:knowledge base 2:caculation 3:Aladdin |
-| adoption_type | The type of adoption | 2:solution 3:command 4:scenario |
-| content | The contents adopted by users | command name/scenario name |
-| arguments | The arguments adopted by users | arguments name |
+
+#### Custom Identification
+| Field          | Content                                                        | Value                                                                              |
+|----------------|----------------------------------------------------------------|------------------------------------------------------------------------------------|
```

Comment:
I believe this document has been added to PR https://github.com/hackathon-cli-recommendation/cli-recommendation/pull/32, right?